src/openforms/tests/e2e/base.py

```diff
@@ -0,0 +1,68 @@
+import os
+from contextlib import asynccontextmanager
+from typing import Literal
+
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.test import override_settings, tag
+from django.urls import reverse
+
+from asgiref.sync import sync_to_async
+from furl import furl
+from playwright.async_api import BrowserType, Page, async_playwright
+
+from openforms.accounts.tests.factories import SuperUserFactory
+
+HEADLESS = "NO_E2E_HEADLESS" not in os.environ
```

Comment:
Isn't this saying "If you have NO_E2E_HEADLESS in your environment, whatever its value, run not headless"?

Reply:
What I meant is that I found a bit confusing that `NO_E2E_HEADLESS=0` will not actually deactivate the non headless mode, so you have to remove the setting from the environment if you have set `NO_E2E_HEADLESS=1`

Reply:
resolved via documentation